libs/designsystem/src/lib/components/toast/services/toast.helper.ts

```diff
@@ -30,9 +29,18 @@ export class ToastHelper {
 
   private getCssClass(messageType: MessageType): string {
     let cssClass = ToastHelper.CSS_CLASS;
-    if (messageType) {
-      cssClass += ' ' + messageType;
+    const type: MessageType =
+      messageType !== 'danger' ? messageType : this.warnAboutDepricatedMessageType();
+    if (type) {
+      cssClass += ' ' + type;
     }
     return cssClass;
   }
```

Comment:
I think we could simplify this a bit, and make it easier to just remove the extra check when we decide the deprecation period is over, like this:

```suggestion
    if (messageType === 'danger') {
      return cssClass += ' ' + 'warning';
    }
    return cssClass += ' ' + messageType;
  }
```

Reply:
Then you just have to return the `cssClass += warning` in the method, or move it to the check.

Reply:
E.g. like this:
```suggestion
    private getCssClass(messageType: MessageType): string {
    let cssClass = ToastHelper.CSS_CLASS;
    if (messageType === 'danger') {
      console.warn(
        `[DEPRECATED] 'danger' message type is deprecated. Use Alerts for critial events. Message type will be set as 'warning'`
      );
      return cssClass += ' ' + 'warning';
    }
    return cssClass += ' ' + messageType;
  }
```